### ISSUE TITLE
PyPI wheel contains old mixin structure

### ISSUE THREAD

**Issue by jmanteau** [jeremyschulman/aio-ipfabric#18]
mixin_configs.py, mixin_inventory.py and mixin_diagrams.py are still present in the PyPI wheel.

Proof:

Install wheel and check -> old mixin_NAME at the root PRESENT
```
❯ pip install aio-ipfabric &&  cd ~/.pye/v/3.8/P/Versions/3.8/lib/python3.8/site-packages/aioipfabric
❯ ls
__init__.py        aiofut.py          base_client.py     consts.py          mixin_configs.py   mixin_inventory.py
__pycache__        api.py             client.py          filters.py         mixin_diagrams.py  mixins
```

Download .tar.gz and check -> old mixin_NAME at the root NOT present
```
❯ cd ~/Downloads/aio-ipfabric-0.6.2 
❯ ls aioipfabric
__init__.py    aiofut.py      api.py         base_client.py client.py      consts.py      filters.py     mixins
```

**Comment by jeremyschulman** [comment]
@jmanteau - you are correct that I *broke* the original structure when I refactored the code in v0.6.0.    I will do a clear install from pip to double-check what is actually wheel installed.  Thank you.

**Comment by jeremyschulman** [comment]
@jmanteau - I installed from pip into a new virtual env, and I do see the same results.  Odd.  I'll fixed this as part of 6.0.3.  Thank you for the issue.

**Comment by jeremyschulman** [comment]
This issue has been resolved in 0.6.3, available in PyPi.
